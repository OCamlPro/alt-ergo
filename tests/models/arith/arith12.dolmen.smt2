(set-option :produce-models true)
(set-option :verify-models false)
(set-logic ALL)
(declare-const x Int)
(assert (<= 0 x 10))
(push 1)
(maximize x)
(check-sat)
(get-model)
(get-objectives)
(pop 1)
(minimize x)
(check-sat)
(get-model)
(get-objectives)
