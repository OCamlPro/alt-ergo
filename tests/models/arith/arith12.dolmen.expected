
unknown
(
  (define-fun x () Int 10)
)
(objectives 
  (x 10))
unknown
(
  (define-fun x () Int 10)
)
(objectives 
  (x 10)
  (x 10)
)